TODO.txt

```diff
@@ -0,0 +1 @@
+the stock task is wrong bc status gets overwritten and we use the 
```

Comment:
I'll be addressing this in my v0 cleanup PR and a note has been made in the v0 testing data issue. Can we delete so we don't forget?